visual-js/visual-cypress/src/types.ts

```diff
@@ -64,11 +74,21 @@ export type VisualCheckOptions = {
   /**
    * The diffing method we should use when finding visual changes. Defaults to DiffingMethod.Balanced
    */
-  diffingMethod?: DiffingMethod;
+  diffingMethod?: `${DiffingMethod}`;
```

Comment:
Note that this is a 'breaking' type change, but since this doesn't work currently I think this is safe to do. Since you cannot import `DiffingMethod` from `@saucelabs/visual` without it blowing up a test due to bundling of modules from tsup.

Reply:
Why not set it to ``DiffingMethod | `${DiffingMethod}` `` like in `ScreenshotMetadata`?

Reply:
`DiffingMethod` doesn't work (you can't import from `@saucelabs/visual` without it breaking) so I didn't want to suggest to TS / IDEs to do so.